go.mod

```diff
@@ -1,30 +1,32 @@
 module github.com/submariner-io/submariner
 
-go 1.13
+go 1.16
 
 require (
-	github.com/cenk/hub v1.0.1 // indirect
+	github.com/cenkalti/backoff/v4 v4.1.2
 	github.com/coreos/go-iptables v0.6.0
-	github.com/ebay/go-ovn v0.1.1-0.20210414223409-7376ba97f8cd
 	github.com/emirpasic/gods v1.18.1
 	github.com/go-ping/ping v0.0.0-20210506233800-ff8be3320020
 	github.com/kelseyhightower/envconfig v1.4.0
 	github.com/onsi/ginkgo v1.16.5
 	github.com/onsi/gomega v1.19.0
+	github.com/ovn-org/libovsdb v0.6.1-0.20220427123326-d7b273399db4
+	github.com/ovn-org/ovn-kubernetes/go-controller v0.0.0-20220427155709-326337d39d6b
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.12.1
 	github.com/submariner-io/admiral v0.13.0-m1
 	github.com/submariner-io/shipyard v0.13.0-m1
 	github.com/uw-labs/lichen v0.1.7
-	github.com/vishvananda/netlink v1.1.0
+	github.com/vishvananda/netlink v1.1.1-0.20210518155637-4cb3795f2ccb
 	golang.org/x/sys v0.0.0-20220114195835-da31bd327af9
 	golang.zx2c4.com/wireguard/wgctrl v0.0.0-20211215182854-7a385b3431de
 	google.golang.org/protobuf v1.27.1
-	k8s.io/api v0.21.0
-	k8s.io/apimachinery v0.21.0
+	k8s.io/api v0.23.3
+	k8s.io/apimachinery v0.23.3
 	k8s.io/client-go v1.5.2
 	k8s.io/klog v1.0.0
-	k8s.io/utils v0.0.0-20210305010621-2afb4311ab10
+	k8s.io/klog/v2 v2.30.0
```

Comment:
I think this klog bump is breaking CI?

```
#11 8.480 [15:49:43.881] [submariner]$ CGO_ENABLED=0 go build -trimpath -ldflags -s -w -X main.VERSION=devel-1a5ac81 -o bin/linux/amd64/submariner-gateway .
#11 80.41 # sigs.k8s.io/controller-runtime/pkg/log
#11 80.41 vendor/sigs.k8s.io/controller-runtime/pkg/log/log.go:50:31: cannot use NullLogger{} (value of type NullLogger) as type logr.Logger in argument to NewDelegatingLogger
#11 80.41 vendor/sigs.k8s.io/controller-runtime/pkg/log/log.go:54:24: cannot use Log (variable of type *DelegatingLogger) as type logr.Logger in variable declaration
#11 80.41 vendor/sigs.k8s.io/controller-runtime/pkg/log/log.go:56:16: assignment mismatch: 1 variable but logr.FromContext returns 2 values
#11 80.41 vendor/sigs.k8s.io/controller-runtime/pkg/log/null.go:30:21: cannot use NullLogger{} (value of type NullLogger) as type logr.Logger in variable declaration
#11 80.41 vendor/sigs.k8s.io/controller-runtime/pkg/log/null.go:48:37: undefined: logr.InfoLogger
#11 80.41 vendor/sigs.k8s.io/controller-runtime/pkg/log/null.go:54:9: cannot use log (variable of type NullLogger) as type logr.Logger in return statement
#11 80.41 vendor/sigs.k8s.io/controller-runtime/pkg/log/null.go:59:9: cannot use log (variable of type NullLogger) as type logr.Logger in return statement
#11 80.41 vendor/sigs.k8s.io/controller-runtime/pkg/log/deleg.go:103:9: cannot use res (variable of type *DelegatingLogger) as type logr.Logger in return statement
#11 80.41 vendor/sigs.k8s.io/controller-runtime/pkg/log/deleg.go:116:9: cannot use res (variable of type *DelegatingLogger) as type logr.Logger in return statement
#11 105.6 make: *** [Makefile:85: bin/linux/amd64/submariner-gateway] Error 2
```

Reply:
yeah we need to keep `logr` at 0.4.0 so we need to keep K8s at 0.21.11 and `klog/v2` at v2.9.0 (or pin it).

Reply:
So the OVN-k Bits require we have k8s 23.3..... Thankfully I was able to bump k8s version (unpinning all pinned versions) without any issues

Reply:
We ran into issues in **submariner-operator** with **lighthouse** bumped to v0.23.5 - we had to pin a bunch of things. We're going to have to do the same here (ie pin k8s 0.21.11). `sigs.k8s.io/controller-runtime` v0.11.2 uses the backwards incompatible `logr` 1.2.0 - we need to directly set or pin it to v0.9.7. Also `github.com/go-logr/logr` pinned to v0.4.0 and `k8s.io/klog/v2` to v2.9.0.